### PR TITLE
feat: add auto-update for python projects

### PR DESCRIPTION
**Pull request by NikoKrismerInnio**

### CODE REVIEW

File: workflow-templates/common-auto-update.yml [hunk 1]
```diff
@@ -115,6 +115,14 @@ jobs:
         if: steps.determine-type-yarn2.outputs.cnt_validations != '0'
         run: echo PROJECT_TYPE=yarn2 >> ${GITHUB_ENV}
 
+      - name: Test project-type (for python)
+        id: determine-type-python
+        run: echo cnt_validations=$(ls ${{ env.TARGET_DIR }}/deploy* | xargs grep -w 'PYTHON_VERSION_FILE' | wc -l) >> ${GITHUB_OUTPUT}
+
+      - name: Set project type to yarn classic
```
Comment (andreasschachinger) [inline workflow-templates/common-auto-update.yml:122]: Isn't the project type set to python?

Comment (NikoKrismerInnio) [reply workflow-templates/common-auto-update.yml]: yes... true. thanks for spotting this

File: workflow-templates/common-auto-update.yml [hunk 2]
```diff
@@ -115,6 +115,14 @@ jobs:
         if: steps.determine-type-yarn2.outputs.cnt_validations != '0'
         run: echo PROJECT_TYPE=yarn2 >> ${GITHUB_ENV}
 
+      - name: Test project-type (for python)
+        id: determine-type-python
+        run: echo cnt_validations=$(ls ${{ env.TARGET_DIR }}/deploy* | xargs grep -w 'PYTHON_VERSION_FILE' | wc -l) >> ${GITHUB_OUTPUT}
+
+      - name: Set project type to python
+        if: steps.determine-type-pyhton.outputs.cnt_validations != '0'
```
Comment (andreasschachinger) [inline workflow-templates/common-auto-update.yml:123]: typo in "pyhton"

Comment (NikoKrismerInnio) [reply workflow-templates/common-auto-update.yml]: oh dear... thanks again

**Review by andreasschachinger — APPROVED** [review]

**Pull request merged**